openssl/src/ssl/mod.rs

```diff
@@ -774,12 +774,12 @@ impl SslContextBuilder {
     /// curve.
     ///
     /// Requires the `v102` feature and OpenSSL 1.0.2.
-    #[cfg(all(feature = "v102", ossl102))]
+    #[cfg(all(feature = "v102", any(ossl102, libressl)))]
```

Comment:
`librev23` corresponding to LibreSSL 2.3 maybe, and similarly for the other versions we support? I'm not sure when `ecdh_auto` was added, but if it's always been around in every release of LibreSSL we can probably then just have an unversioned `libre` feature.

The reason I'm worrying about this is that LibreSSL mirrors OpenSSL 1.0.1 APIs, but then apparently with some other additions like this, so it doesn't really correspond to existing feature.